commit f31e5b71d325bc3bce98f3460c101ceb36b5cb60
Author:     nvkevlu <55759229+nvkevlu@users.noreply.github.com>
AuthorDate: Fri Mar 25 17:27:36 2022 -0400
Commit:     GitHub <noreply@github.com>
CommitDate: Fri Mar 25 14:27:36 2022 -0700

    move job def related into apis, and add get_apps (#350)
---
 nvflare/private/fed/server/server_engine.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nvflare/private/fed/server/server_engine.py b/nvflare/private/fed/server/server_engine.py
index 1573ecbe..b8385576 100644
--- a/nvflare/private/fed/server/server_engine.py
+++ b/nvflare/private/fed/server/server_engine.py
@@ -30,13 +30,13 @@ from nvflare.apis.client import Client
 from nvflare.apis.fl_constant import FLContextKey, MachineStatus, ReservedTopic, ReturnCode, SnapshotKey
 from nvflare.apis.fl_context import FLContext
 from nvflare.apis.fl_snapshot import FLSnapshot
+from nvflare.apis.impl.job_def_manager import SimpleJobDefManager
 from nvflare.apis.shareable import Shareable, make_reply
 from nvflare.apis.study_manager_spec import StudyManagerSpec
 from nvflare.apis.utils.fl_context_utils import get_serializable_data
 from nvflare.apis.workspace import Workspace
 from nvflare.app_common.storages.filesystem_storage import FilesystemStorage
 from nvflare.fuel.hci.zip_utils import zip_directory_to_bytes
-from nvflare.mt.job_def_manager import SimpleJobDefManager
 from nvflare.private.admin_defs import Message
 from nvflare.private.fed.server.server_json_config import ServerJsonConfigurator
 from nvflare.widgets.info_collector import InfoCollector
